### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by dang1840**
1. I made a few minor changes to test thing out.
2. I commented out couple of the @import in styles.css. They are not working correctly on my end.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -94,5 +94,8 @@
         }
       }
     }
+  },
+  "cli": {
+    "analytics": false
```
Comment (melo-velo) [inline angular.json:99]: Not sure what this change is, but it looks fine

File: src/app/app.component.html [hunk 2]
```diff
@@ -3,7 +3,7 @@ <h1>{{pageHeader}}</h1>
   <div>Project members: </div>
   <ul>
     <li>Martin Jaakola</li>
-    <li>Dang Vu Quang</li>
+    <li>Dang Dang</li>
```
Comment (melo-velo) [inline src/app/app.component.html:6]: Do you prefer to be called/referred to this way?

File: src/app/app.component.ts [hunk 3]
```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  pageHeader = 'Itemize';
+  public pageHeader = 'Itemize';
```
Comment (melo-velo) [inline src/app/app.component.ts:9]: This is actually unnecessary in Typescript

**Comment by melo-velo** [comment]
Looks good to me

**Review by melo-velo — APPROVED** [review]

File: angular.json (hunk 1 above)
Comment (dang1840) [reply angular.json:99]: This will stop  Angular asking permission to collect data

Comment (dang1840) [reply angular.json:99]: This means we deny Angular permission to collect data

**Pull request merged**